src/nsls2api/api/v1/admin_api.py

```diff
@@ -55,14 +55,8 @@ async def generate_user_apikey(username: str):
 
 @router.post("/admin/proposal/generate-test")
 async def generate_fake_proposal(
-    use_real_people: bool = False,
+    add_specific_user: str,
```

Comment:
Shouldn't this be optional as we don't want to force people to enter something here

Reply:
Addressed in https://github.com/NSLS2/nsls2api/pull/97/commits/9b30ad4d2332d40345da8b7b5917dc46f52ca478